tutorials/01-gaussian-mixture-model/01_gaussian-mixture-model.jmd

```diff
@@ -103,63 +101,68 @@ We are also calling `MCMCThreads` to generate multiple chains, particularly so w
 
 ```julia
 gmm_sampler = Gibbs(PG(100, :k), HMC(0.05, 10, :μ1, :μ2))
-tchain = sample(gmm_model, gmm_sampler, MCMCThreads(), 100, 3);
+tchain = mapreduce(c -> sample(gmm_model, gmm_sampler, 100), chainscat, 1:3);
```

Comment:
If you want to sample in serial (no multithreading) it is recommended/standard to use
```suggestion
tchain = sample(gmm_model, gmm_sampler, MCMCSerial(), 100, 3);
```